swagger.yaml

```diff
@@ -1,75 +1,83 @@
 swagger: '2.0'
+
 info:
-  title: Web API Skeleton
-  description: Skeleton for Dropwizard Web APIs
-  version: '0.0'
-  license:
-    name: GNU Affero General Public License Version 3
-    url: http://www.gnu.org/licenses/agpl-3.0.en.html
-externalDocs:
-  description: GitHub Repository
-  url: https://github.com/osu-mist/web-api-skeleton
-host: localhost
-schemes:
-  - http
-basePath: /api/v0
-consumes:
-  - text/plain
+  version: "0.0.1"
+  title: OSU Dining Balances API
```

Comment:
I'm ok with this name but then we should change the name of the git repo. Which name do you want to go with
?